de.metas.contracts/src/test/java/de/metas/contracts/impl/AbstractFlatrateTermTest.java

```diff
@@ -325,8 +338,8 @@ protected I_C_Flatrate_Term createFlatrateTerm(@NonNull final I_C_Flatrate_Condi
 		contract.setDropShip_BPartner(getBpartner());
 		contract.setDropShip_Location(bpLocation);
 		contract.setDropShip_User(user);
-		contract.setPriceActual(BigDecimal.valueOf(2));
-		contract.setPlannedQtyPerUnit(BigDecimal.ONE);
+		contract.setPriceActual(PRICE);
```

Comment:
please rename to `QTY_TEN` and `PRICE_ONE`..it makes it more readable when you use them somewhere else

Reply:
agree